src/pages/AdminTemplates/AdminTemplates.scss

```diff
@@ -0,0 +1,169 @@
+@import "scss/constants.scss";
+
+.templates-page {
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 5rem;
+
+  &__nav-buttons {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 80%;
+
+    &--cancel {
+      padding: 0.5rem 1.5rem;
+      border-radius: 22px;
```

Comment:
Can we extract/use this as a variable from `scss/constants` in a similar way to how it's defined in https://doesitsparkle.slack.com/archives/C01JHLMSFND/p1619732604371600?thread_ts=1619732501.371300&cid=C01JHLMSFND

![image](https://user-images.githubusercontent.com/753891/116633788-bfeef400-a99d-11eb-8c78-a7b1a00238b5.png)

Reply:
(and all of the other places this is used)